packages/nimble-components/src/utilities/models/update-tracker.ts

```diff
@@ -0,0 +1,14 @@
+import { Tracker } from './tracker';
+
+/**
+ * Generic Update Tracker Utility which extends Tracker Utility with update logic
+ */
+export abstract class UpdateTracker<
+    TrackedProperties extends readonly string[]
+> extends Tracker<TrackedProperties> {
+    public constructor(requiredUpdates: TrackedProperties) {
+        super(requiredUpdates);
+    }
```

Comment:
Constructor isn't doing anything anymore, think you can leave it off and calls will go right to the parent